rmf_site_format/src/legacy/building_map.rs

```diff
@@ -85,6 +85,16 @@ impl BuildingMap {
                 camera.yaw -= delta_yaw;
             }
 
+            for light in &mut level.lights {
+                let p = tf.transform_point2(DVec2::new(
+                    light.pose.trans[0] as f64,
+                    light.pose.trans[1] as f64,
+                ));
+                light.pose.trans[0] = p.x as f32;
+                light.pose.trans[1] = -p.y as f32;
+                light.pose.rot.apply_yaw(Angle::Rad(-delta_yaw as f32));
+            }
```

Comment:
I don't know how deep we want to go into adding features to the legacy format but a yaw only angle doesn't sound quite sufficient for lights since all the current lights have symmetry on yaw and angle only has an impact on lights if roll or pitch are changed

Reply:
This line of code is only for modifying the user-specified rotation based on how the levels were moved during the alignment process.

I'll tweak the example in the office demo to show a light with some pitch.

Reply:
Now there's an example of [a spot light with some roll](https://github.com/open-rmf/rmf_site/blob/ce0fa6e9fe1c535445e50cdb61cff3aea52433fb/assets/demo_maps/office.building.yaml#L138) in the `.building.yaml`.